VMS/VMS.csproj

```diff
@@ -47,7 +47,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Resources\src\common\" />
```

Comment:
why do we have to delete this line?

Reply:
Easy peasy, if we don't delete this line, we cannot put file into /common folder.
By the way, this is done by VS, not by me :D

Reply:
Ok i see